Jenkinsfile_CNP

```diff
@@ -212,6 +214,7 @@ withPipeline(type, product, component) {
   onPR {
     env.ENVIRONMENT="preview"
     env.CASE_API_URL = "http://sptribs-case-api-pr-${CHANGE_ID}-java"
+    env.TEST_URL = "http://sptribs-case-api-pr-${CHANGE_ID}-java"
```

Comment:
Do we need this one as well? Is it worth pointing whichever var is using TEST_URL to use CASE_API_URL instead since they are the same?

Reply:
TEST_URL is not needed so removed